pkgs/top-level/all-packages.nix

```diff
@@ -23258,8 +23260,10 @@ in
 
   fts = if stdenv.hostPlatform.isMusl then netbsd.fts else null;
 
-  inherit (recurseIntoAttrs (callPackages ../os-specific/bsd { }))
-          netbsd;
+  netbsd = callPackages ../os-specific/bsd/netbsd {};
```

Comment:
This removed `recurseIntoAttrs` and thus the packages aren't built by Hydra anymore: https://hydra.nixos.org/eval/1512985?filter=&compare=1512944&full=#tabs-removed  Was that intentional?